src/lib/graphqlErrorHandler.ts

```diff
@@ -81,7 +81,11 @@ const reportErrorToSentry = (
   })
 }
 
-type WriteablePartial<T> = { -readonly [P in keyof T]+?: T[P] }
+export type WriteablePartial<T> = { -readonly [P in keyof T]+?: T[P] }
+
+export type GraphQLErrorHandler = (
+  topLevelError: GraphQLError
+) => WriteablePartial<GraphQLFormattedError>
```

Comment:
The writeable part of this should probably remain private to the module scope, as it’s only needed for the newly code that assigns to the object.

Reply:
Yep, that's a left over from when I was exporiting it (before I exported the `GraphQLErrorHandler` explicitly). Good call.